src/front_end/index.html

```diff
@@ -31,7 +31,7 @@ <h1>Logged in as Super User (admin) at Registration desk</h1>
         <div class="card-body">
           <h5 class="card-title">Find patient record</h5>
           <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
-          <a id="test" href="patientVisit.html" class="btn btn-secondary">Go</a>
```

Comment:
(ik mag geen comment toevoegen op niet gehighlighte code, dus dan maar zo). er staat nog een style tag bovenin voor style.css. als je die weghaalt (bij alle html files) dan ben je van de style.css error af

Reply:
en lastmeasurement.js is leeg?

Reply:
also: patientoverview.js en patientvitals.js missen style import